test/crash/src/test/java/io/narayana/lra/arquillian/LRACoordinatorRecoveryIT.java

```diff
@@ -110,14 +110,14 @@ public void before() {
 
         client = ClientBuilder.newClient();
         lraClient = new NarayanaLRAClient();
-
+        // Cleans Object Store in case there are transactions from previous tests
+        clearRecoveryLogFromFS();
         for (Map.Entry<String, String> entry : containerDeploymentMap.entrySet()) {
             startContainer(entry.getKey(), entry.getValue());
         }
 
-        // Cleans Object Store in case there are transactions from previous tests
         clearRecoveryLogFromObjectStore();
-        clearRecoveryLogFromFS();
```

Comment:
I am OK with this being moved before the start of the container, it seems clean to me to do that. I would be tempted to fail the test if there was something in the objectstore after starting the new test (or make the assumption there should be nothing and remove the call to clearRecoveryLogFromObjectStore but these are remarks rather than requests for change

Reply:
Thanks Tom, I think this is related to my previous comment. In the CI we have the --fail-at-end flag when running tests. So we want to make sure that a failure is not impacting the following tests.

Reply:
Anyway, it was a remark so I will resolve this

Reply:
Actually, I am not mandating the fail checks being added but to add readability of my review I will unresolve it but it does not need addressing IMO